drivers/spi/spi_ll_stm32.c

```diff
@@ -98,14 +97,20 @@ static void spi_stm32_shift_m(SPI_TypeDef *spi, struct spi_stm32_data *data)
 	if (SPI_WORD_SIZE_GET(data->ctx.config->operation) == 8) {
 		rx_frame = LL_SPI_ReceiveData8(spi);
 		if (spi_context_rx_on(&data->ctx)) {
-			UNALIGNED_PUT(rx_frame, (u8_t *)data->ctx.rx_buf);
```

Comment:
Why not using `&& data->ctx.rx_buf` as you did for tx?
It makes both change and code lighter.
I guess this is a leftover from initial commit. Latest code allows you to do it now.

Reply:
I'll fix it.